00-intro.md

```diff
@@ -64,13 +64,13 @@ Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
 Using Bash or any other shell
-sometimes feels more like programming than like using a mouse.
```

Comment:
This is "sometimes feels" to make a distinction between the shell and programming proper, I believe.